core/operation/src/main/java/uk/gov/gchq/gaffer/named/operation/AddNamedOperation.java

```diff
@@ -54,9 +54,11 @@
 @Since("1.0.0")
 @Summary("Adds a new named operation")
 public class AddNamedOperation implements Operation, Operations<Operation> {
+
     @Required
     private String operations;
     private String operationName;
+    private String label;
```

Comment:
It might be better to have a list of labels so that named operations can have more than one.

Reply:
Ah yes it did cross my mind. Having an Operation linked to many groups would be a better feature - and a lot more flexible - I'll make the change thanks

Reply:
Done